R/lancelot.R

```diff
@@ -740,6 +740,14 @@ lancelot_index <- function(info) {
                   deaths_carehomes_inc = index[["D_carehomes_inc"]],
                   deaths_comm_inc = index[["D_comm_inc"]],
                   deaths_hosp_inc = index[["D_hosp_inc"]],
+                  deaths_hosp_inc_0_49 = index[["D_hosp_inc_0_49"]],
```

Comment:
I think we should change ordering to `deaths_hosp_0_49_inc` (and also `D_hosp_0_49_inc` in the odin code), it's consistent with how `_inc` always comes last (e.g. `sympt_cases_15_24_inc`)

Reply:
Done